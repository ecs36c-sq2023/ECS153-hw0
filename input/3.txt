Rectangle
5 
3 3
2 4
1 8
4 4
4 2
