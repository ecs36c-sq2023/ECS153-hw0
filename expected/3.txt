4 4
3 3
4 2
1 8
2 4
